game-core/src/main/java/games/strategy/triplea/ui/ObjectivePanel.java

```diff
@@ -196,17 +196,16 @@ private void setObjectiveStats() {
 
         // find which section
         boolean found = false;
-        if (sections.containsKey(player.getName())) {
-          if (sections.get(player.getName()).contains(key.get(1))) {
-            final Map<ICondition, String> map = statsObjectiveUnsorted.get(player.getName());
-            if (map == null) {
-              throw new IllegalStateException(
-                  "objective.properties group has nothing: " + player.getName());
-            }
-            map.put(condition, value);
-            statsObjectiveUnsorted.put(player.getName(), map);
-            found = true;
+        if (sections.containsKey(player.getName())
+            && sections.get(player.getName()).contains(key.get(1))) {
```

Comment:
Just a thought:
```java
sections.getOrDefault(player.getName(), List.of()).contains(key.get(1))
```

would be a shorter variant of the if statement, I haven't convinced myself that it's clearly better though

Reply:
A more radical change introduces unwanted risk and perhaps should arguably be tested. I'm not saying I disagree, only that such a change should be done as it's own PR where we make that one change or a set of similar changes and ensure it is adequately tested.

Reply:
A multimap might be what we want. Having a person reading code realize that the later execution is a no-op because we default to an empty list is a bit indirect. It is arguably simpler than the series of if statements, I agree that is debatable.